src/components/Header/components/NavButtonsRightCorner/index.jsx

```diff
@@ -50,6 +50,16 @@ const GordonNavButtonsRightCorner = ({ onClose, openDialogBox, open, anchorEl })
     />
   );
 
+  const transcriptButton = !isAuthenticated ? (
+    <GordonNavButton
+      unavailable={!isOnline ? 'offline' : null}
+      onLinkClick={onClose}
+      openUnavailableDialog={openDialogBox}
+      linkName={'Experience Transcript'}
+      linkPath={'/transcript'}
+    />
+  ) : null;
```

Comment:
This is shorthand for the same thing:
```suggestion
  const transcriptButton = !isAuthenticated && (
    <GordonNavButton
      unavailable={!isOnline ? 'offline' : null}
      onLinkClick={onClose}
      openUnavailableDialog={openDialogBox}
      linkName={'Experience Transcript'}
      linkPath={'/transcript'}
    />
  );
```

Reply:
Technically, these are not equivalent. If the user is authenticated, the original code sets `transcriptButton` to `null`, whereas your suggestion will set `transcriptButton` to `false`. However, because JSX treats `null` and `false` the same way for rendering (i.e. "Don't render anything for this element"), they are functionally equivalent. I just wanted to clarify that this isn't some JS voodoo, it's a shortcut specific to how JSX renders HTML from JS values.

Reply:
So yes, it's shorthand for the same thing with JSX.

Thanks for the helpful details!